src/lib/serviceRoutinesV2/putEntityAttributeValue.cpp

```diff
@@ -79,6 +81,15 @@ std::string putEntityAttributeValue
     }
   }
 
+  if (ciP->httpStatusCode == SccConflict)
+  {
+    ErrorCode   ec("TooManyResults", "There is more than one entity that match the update. Please refine your query.");
```

Comment:
More 'similar  texts':

    putEntity.cpp:                 OrionError orionError(SccConflict, "There is more than one entity that match the update. Please refine your query.");
    patchEntity.cpp:               OrionError orionError(SccConflict, "There is more than one entity that match the update. Please refine your query.");
    ContextAttributeResponse.cpp:  alarmMgr.badInput(clientIp, "more than one context element found in this query - selecting the first one");
    ContextElementResponse.cpp:    alarmMgr.badInput(clientIp, "more than one context element found the this query - selecting the first one");
    MongoCommonUpdate.cpp:         buildGeneralErrorResponse(ceP, NULL, responseP, SccConflict, "There is more than one entity that match the update. Please refine your query.");